play-routes-compiler/src/main/scala/CommandLinePlayRoutesCompiler.scala

```diff
@@ -63,6 +66,12 @@ object CommandLinePlayRoutesCompiler {
     }.text("Whether the reverse router should be namespaced. Useful if you have many routers that use the same actions.")
   }
 
+  private def stripHeader(path: String): Unit = {
+    val lines = Source.fromFile(path).getLines
+    val sansHeader = lines.span(line => line.matches("//.*|\\s*"))._2
```

Comment:
the regex should probably have a "^" at the beginning to anchor to the beginning of the line. We don't want to accidentally remove code that has a comment at the end

Reply:
I believe `matches` only returns true if it's a perfect match, but I can add that just in case